### ISSUE TITLE
haarcascade_frontalface_default.xml (No such file or directory)

### ISSUE THREAD

**Issue by ltsiemens** [crockpotveggies/tinderbox#47]
Tinderbox seems to be running, but I'm getting a consistent error. I'm currently on Ubuntu 14.04 if that makes a difference.

[debug] application - [tinderbot] New task received for supervision
[info] application - [tinderbot] Received command: idle
[debug] application - [tinderbot] Starting new facial analysis task for 547a75dd579794bd5b1da53b.
[error] application - [tinderbot] Retrying a troubled supervised task: 
haarcascade_frontalface_default.xml (No such file or directory)
[info] application - [tinderbot] Received command: run
[info] application - [tinderbot] Received command: run
[debug] application - [tinderbot] Starting new facial analysis task for 547a75dd579794bd5b1da53b.
[ERROR] [02/22/2015 18:01:21.564] [application-akka.actor.default-dispatcher-2] [akka://application/user/TinderBot/BotSupervisor/$+g] haarcascade_frontalface_default.xml (No such file or directory)
java.io.FileNotFoundException: haarcascade_frontalface_default.xml (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.<init>(FileInputStream.java:146)
	at java.io.FileInputStream.<init>(FileInputStream.java:101)
	at jviolajones.Detector.<init>(Detector.java:33)
	at utils.face.FacialDetection.detectFaces(FacialDetection.scala:26)
	at utils.face.FacialDetection.extractFaces(FacialDetection.scala:39)
	at models.bot.tasks.FacialAnalysisTask$$anonfun$receive$1$$anonfun$applyOrElse$2.apply(FacialAnalysisTask.scala:39)
	at models.bot.tasks.FacialAnalysisTask$$anonfun$receive$1$$anonfun$applyOrElse$2.apply(FacialAnalysisTask.scala:38)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244)
	at scala.collection.AbstractTraversable.map(Traversable.scala:105)
	at models.bot.tasks.FacialAnalysisTask$$anonfun$receive$1.applyOrElse(FacialAnalysisTask.scala:38)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:498)
	at akka.actor.ActorCell.invoke(ActorCell.scala:456)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:237)
	at akka.dispatch.Mailbox.run(Mailbox.scala:219)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)

**Comment by crockpotveggies** [comment]
A couple questions:

1. Did you start the app using the provided script?
2. Did you make any changes and build it yourself?

**Comment by ltsiemens** [comment]
1. I did run the app using the provided script.
2. I made a change to the Fun Messages file, and tried to rebuild it. An error came up at that time, but I don't remember exactly what it said. Everything seemed to be ok though.

I ended up just starting from scratch, didn't touch the messages file, and it seems to have fixed everything. Things are working fine.

I'll see what happens when I go to edit the message file in the future.
